### ISSUE TITLE
🔨 Mog Tablets

### ISSUE THREAD

**Issue by zach2good** [DerpyProjectGroup/topaz#2095]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)

## Describe the feature

Implement Mog Tablets. 

This is interesting because there is a lot of cross-zone and possible cross-process communication, for which we don't have any mechanisms yet.

I've found the documentation to be a bit lacking, since this isn't a feature many people do, aside from by accident.

We've got ZMQ sending strings between processes that will be executed by the receiver's Lua state, there's just a little more plumbing to do, then some research and then the implementation can go ahead.

This is also a good opportunity for us to use volatile server vars and caching from this PR: https://github.com/LandSandBoat/server/pull/2067

### Docs

- http://www.playonline.com/ff11us/guide/mogtab/index.html
- http://chrysaliswiki.com/activity:kupowers-xi
- http://wiki.ffo.jp/html/17461.html
- https://ffxiclopedia.fandom.com/wiki/Mog-Tablet
- https://www.bg-wiki.com/ffxi/The_Mog_Tablet_Quest
- https://www.ffxiah.com/forum/topic/54594/mog-tablets-being-found-with-no-one-there

### PRs

- https://github.com/LandSandBoat/server/pull/1123

**Comment by The-Aerec** [comment]
This would be a good candidate for moving to the planned: 'World_Server' exe? (Whenever that comes along)

**Comment by zach2good** [comment]
Not really, it's just a couple of very infrequent announcements, world server would be overkill for this

**Comment by RunAwayDSP** [comment]
Ive been trying to capture these on retail with no success but i do remember se had added in a radius check before it would "spawn" (when if first released you could find its pos from anywhere in zone) It may have even been improved upon more, but I remeber there had to be someone within sight radius for it to even appear on widescan (Scanning tools could even spot it if other players got close). Considering this im certain a region check would be needed just to "spawn" the tablet in the pre-determined zone/pos
